docs/modules/ROOT/partials/drain-node.adoc

```diff
@@ -0,0 +1,144 @@
+* If you are working on a production cluster, you need to *schedule the node drain for the next maintenance.*
+* If you are working on a non-production cluster, you may *drain and remove the node immediately.*
+
+=== Schedule node drain (production clusters)
```

Comment:
The only how-tos where we actually need to schedule the drain for a maintenance are the generic `remove-node` ones. The storage node how-tos never require this.

Reply:
Good to know, thanks